app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -43,14 +43,35 @@ function buildArticleHTML(article) {
     var rc = article.positive_reactions_count || article.reactions_count
     var reactionsCountHTML = ''
     if ((rc || '0') > 0 && article.class_name != "User") {
-      var reactionsCountHTML = '<div class="article-engagement-count reactions-count"><a href="'+article.path+'"><img src="<%= asset_path("reactions-stack.png") %>" alt="heart" /><span id="engagement-count-number-'+article.id+'" class="engagement-count-number">'+(rc || '0')+'</span></a></div>'
+      var reactionsCountHTML = '<a href="'+article.path+'" class="crayons-story__details__item"><svg class="crayons-icon" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M18.884 12.595l.01.011L12 19.5l-6.894-6.894.01-.01A4.875 4.875 0 0112 5.73a4.875 4.875 0 016.884 6.865zM6.431 7.037a3.375 3.375 0 000 4.773L12 17.38l5.569-5.569a3.375 3.375 0 10-4.773-4.773L9.613 10.22l-1.06-1.062 2.371-2.372a3.375 3.375 0 00-4.492.25v.001z"/></svg>'+(rc || '0')+'<span class="hidden s:inline">&nbsp;comments</span></a>'
     }
     var picUrl = article.user.profile_image_90
     var profileUsername = article.user.username
-    var orgHeadline = "";
+    var forOrganization = "";
+    var organizationLogo = "";
+    var organizationClasses = "crayons-avatar--l";
     if (article.organization && !document.getElementById("organization-article-index")) {
-      orgHeadline = '<div class="article-organization-headline"><a class="org-headline-filler" href="/'+article.organization.slug+'"><img alt="'+article.organization.name+' logo" src="'+article.organization.profile_image_90+'" loading="lazy">'+article.organization.name+'</a></div>'
+      var organizationLogo = '<a href="/'+article.organization.slug+'" class="crayons-logo crayons-logo--l"><img alt="'+article.organization.name+' logo" src="'+article.organization.profile_image_90+'" class="crayons-logo__image" loading="lazy"/></a>';
+      var forOrganization = '<span><span class="crayons-story__tertiary fw-normal"> for </span><a href="/'+article.organization.slug+'" class="crayons-story__secondary fw-medium">'+article.organization.name+'</a></span>';
+      var organizationClasses = "crayons-avatar--s absolute -right-2 -bottom-2 border-solid border-2 border-base-inverted";
     }
+
+    var meta = '<div class="crayons-story__meta">\
```

Comment:
I'm basically combining author with organization, the same way as I have in preact component.

![image](https://user-images.githubusercontent.com/108287/78879596-62b39800-7a54-11ea-9b90-c3f695154ebf.png)